gridstatus/ercot_api/ercot_api.py

```diff
@@ -19,6 +19,22 @@
 TOKEN_URL = "https://ercotb2c.b2clogin.com/ercotb2c.onmicrosoft.com/B2C_1_PUBAPI-ROPC-FLOW/oauth2/v2.0/token"  # noqa
 BASE_URL = "https://api.ercot.com/api/public-reports"
 
+# How long a token lasts for before needing to be refreshed
+TOKEN_EXPIRATION_SECONDS = 3600
+
+# Number of historical links to fetch at once. The max is 1_000
+DEFAULT_HISTORICAL_SIZE = 1_000
+
+# Number of results to fetch per page. It's not clear what the max is (1_000_000 works)
+DEFAULT_PAGE_SIZE = 250_000
```

Comment:
I upped the page size because there does not seem to be a maximum and increasing generally reduces overall querying time.